modules/ossm-tutorial-bookinfo-install.adoc

```diff
@@ -74,25 +81,25 @@ You need cluster-admin rights to edit the Istio Service Mesh Member Roll.
 ----
 $ oc -n <control_plane_project> patch --type='json' smmr default -p '[{"op": "add", "path": "/spec/members", "value":["'"bookinfo"'"]}]'
 ----
-
++
 . Click *Create* to save the updated Service Mesh Member Roll.
 
 . From the CLI, deploy the Bookinfo application in the _`bookinfo`_ project by applying the `bookinfo.yaml` file:
 +
 
-[source,bash,subs="attributes"]
+[source,terminal]
 ----
 $ oc apply -n bookinfo -f https://raw.githubusercontent.com/Maistra/istio/maistra-{MaistraVersion}/samples/bookinfo/platform/kube/bookinfo.yaml
 ----
-
++
 . Create the ingress gateway by applying the `bookinfo-gateway.yaml` file:
 +
 
-[source,bash,subs="attributes"]
+[source,terminal]
```

Comment:
@JStickler Can you change this back to `[source,bash,subs="attributes"]`. It enables the version substitution in the command.